src/main/java/seedu/manager/command/AddCommand.java

```diff
@@ -1,20 +1,29 @@
 package seedu.manager.command;
 
+import seedu.manager.event.EventList;
+
 /**
  * Represents an executable add command
  */
 public class AddCommand extends Command {
     public static final String COMMAND_WORD = "add";
-    private static final String ADD_MESSAGE = "The following event has been added to the event list: %1$s";
+    public static final String ADD_MESSAGE = "Event added successfully";
+    protected String eventName;
+    protected EventList events;
+
+    public AddCommand(String desc, EventList events) {
+        this.eventName = desc;
+        this.events = events;
+    }
 
     /**
-     * Implement the adding command
-     * Returns a command output with a successful adding message
+     * Returns a command output with an add message
      *
-     * @return The command output with a successful adding message
+     * @return The command output with an add message
      */
-    @Override
     public CommandOutput execute() {
+        this.events.addEvent(this.eventName);
+        this.events.getList();
         return new CommandOutput(ADD_MESSAGE, false);
     }
```

Comment:
Following on from the comment made at getList(), I think the result from that method could be concatenated with the add message, then the resulting string passed as the output message of the CommandOutput object?